sw/airborne/modules/guidance/gvf/gvf.c

```diff
@@ -457,3 +496,259 @@ bool gvf_sin_wp_alpha(uint8_t wp, float alpha, float w, float off, float A)
   return true;
 }
 
+
+#elif defined(ROVER_FIRMWARE) // ROVER TRAJECTORIES
```

Comment:
if more firmware specific code should be added, consider making a separated C file for each of them

Reply:
Okay, we'll keep these conditionals for now, but we'll consider your suggestion in a next future.